src/main/resources/anniv5Resources/localization/eng/darksoulspack/Cardstrings.json

```
@@ -0,0 +1,44 @@
+{
+  "${ModID}:MorionBlade": {
+    "NAME": "Morion Blade",
+    "DESCRIPTION": "Deal !D! damage. NL Repeat for each unique debuff you have."
+  },
+  "${ModID}:Bloodlust": {
+    "NAME": "Bloodlust",
+    "DESCRIPTION": "Deal !D! damage. Apply 1 Vulnerable to EVERYONE."
```

Could use a NL between these effects.
```suggestion
    "DESCRIPTION": "Deal !D! damage. NL Apply 1 Vulnerable to EVERYONE."
```